### ISSUE TITLE
[Whitelist] Congo (317530)

### ISSUE THREAD

**Issue by liberodark** [ValveSoftware/Proton#1216]
Distro: Manjaro XFCE 17.1.12
GPU: GTX 1060, Nvidia proprietary driver 396.54
Resolution: 1920x1080
RAM: 8GB
Proton 3.7-5 Beta

All is fine and Xbox Controller work !

**Comment by gudvinr** [comment]
Doesn't work with `Radeon Vega M` and `Proton 3.16-4 beta`.
Failed at launch with `DX11 feature level 10.0 is required to run the engine`.

**Comment by kisak-valve** [comment]
Hello @gudvinr, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box. Also, please confirm that any vulkan application like `vulkaninfo` works.

**Comment by gudvinr** [comment]
@kisak-valve Yep, `vulkaninfo` works. Also `PROTON_USE_WINED3D11=1` helps but it probably doesn't use vulkan. Also KF2 (232090) which uses DX11 by default works fine too.
Logs are attached.

[steam-317530.log](https://github.com/ValveSoftware/Proton/files/2665198/steam-317530.log)

**Comment by kisak-valve** [comment]
The log hints that your issue might be PhysX related. Possibly related: https://github.com/ValveSoftware/Proton/issues/165#issuecomment-416086893
